npm/cypress-schematic/package.json

```diff
@@ -7,29 +7,28 @@
     "build": "tsc -p tsconfig.json",
     "build:watch": "tsc -p tsconfig.json --watch",
     "lint": "eslint --ext .ts,.json, .",
-    "test": "mocha -r @packages/ts/register --reporter mocha-multi-reporters --reporter-options configFile=../../mocha-reporter-config.json src/**/*.spec.ts"
+    "test": "vitest run --no-file-parallelism"
```

Comment:
I updated `cypress-schematic` to use `vitest` as the mocking library is much easier to use and it can run ts files natively. I wound up not needing the mocking but still think this a good addition